contrib/emptty/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "emptty"
+pkgver = "0.12.1"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = [
+        "go"
+]
+makedepends = [
+        "libx11-devel",
+        "linux-pam-devel"
+]
+pkgdesc = "Dead simple CLI Display Manager on TTY"
+maintainer = "Michal Tvrznik <emporeor@gmail.com>"
+license = "MIT"
+url = "https://github.com/tvrzna/emptty"
+source = f"https://github.com/tvrzna/emptty/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "0220230cdee3dedd7ccd6f2d81c57b29e2169159c8e28263d58abaf20093ce23"
+env = {"CGO_ENABLED": "1"}
+
+
+def post_install(self):
+    self.install_file(src="res/pam", dest="usr/lib/pam.d/", name="emptty")
+    self.install_file("res/conf", "etc/emptty/")
+    self.install_svscript("res/dinit-script", "emptty")
```

Comment:
```suggestion
```

Reply:
Done, original script was made to do `sh -l` to provide correct `PATH` environment variable in different distro.